app/src/main/res/values/colors.xml

```diff
@@ -26,7 +26,9 @@
     <color name="indicator_dot">#888</color>
     <color name="indicator_dot_selected">@color/accent</color>
 
-    <color name="hint_text">#666</color>
+    <color name="hint_text">#555</color>
+    <color name="icon_selection_title_text">#000</color>
+    <color name="icon_selection_description_text">#000</color>
```

Comment:
The styles are called `ImageSelectTitle` and `ImageSelectDescription`, so I'd suggest using

```suggestion
    <color name="image_select_title_text">#000</color>
    <color name="image_select_description_text">#000</color>
```

Reply:
I even thought about renaming `ImageSelectTitle` `ImageSelectDescription` styles. It seems that this is used only with svg illustrations.

image_select_description_text seems to be about
![screen](https://user-images.githubusercontent.com/899988/135133901-31f4d3c9-f636-4315-8a4f-039763fd33da.png)
that is using a separate style (`@style/ImageSelectLabel`)

Though, in the end bridge quest now also have svg illustrations, and `ImageSelectTitle` `ImageSelectDescription` could be used with tiny photos (in theory at least)

------

`title_text_next_to_image` `TitleNextToImage` + `description_text_next_to_image` `TitleNextToImage` ?

Reply:
> `title_text_next_to_image` `TitleNextToImage` + `description_text_next_to_image` `DescriptionNextToImage` ?

That sounds very clear for me, so :+1:

Reply:
Done, I will keep unresolved in case @westnordost is not a fan of such change.